about.html

```diff
@@ -0,0 +1,220 @@
+            skills and portfolio you need to get hired as a software engineer.
+            The online coding bootcamp offers experienced instructors, in-depth
+            career services, outstanding projects, and more. <br />
+            A certificate for taking the training is prepared by the Boonie
+            company.
+            <br />
+          </p>
+          <div class="about-questions-box">
+            <span class="about-questions">
+              Please contact us per Email for any further questions about the
+              training</span
+            >
+            <span class="about-gmail">Boonie2022@blo.edu.ar</span>
+          </div>
+        </div>
+      </section>
+      <section class="the-logo-gran">
+        <div class="the-logo-sec desktop-pad-version">
+          <h2 class="title-logo-about the-underline">
+            Boonie Bootcamp training 2022 Boonie Boot-Logo
+          </h2>
+          <p class="para-logo-about">
+            The Boonie boot-logo of Web development short-term training 2021 is
+            decided through the logo competetion from 7, march to 10, June. That
+            way, after online voting and the selection committes judgin process,
+            the boot-logo from Matias fenger won the prestiguos prize. Lastly,
+            and after consultation with Mr. Fenger we optimized the prize
+            winning logo for this year's training (2022).
+          </p>
+          <p class="para-logo-about-2">
+            The Boot-log of Software Web development training 2022 is decided by
+            the last year winner.
+          </p>
+          <div class="box-img-logo-container">
+            <img src="./IMG/organico-blanco-y-negro.png" alt="" />
+          </div>
+        </div>
+      </section>
+      <section class="past-bootcamp-exp">
+        <div id="sec-past-events" class="wrapper desktop-pad-version">
+          <h2 class="section-heading title-past-bootcamp the-underline">
+            Past Bootcamps
+          </h2>
+          <p>
+            We are not the first and the only! Discover more events and
+            information below:
+          </p>
+
+          <div class="other-programs">
+            <a href="https://tehransfd.ir" class="program">
+              <img
+                src="./IMG/Past-Bootcamps/women-looking-at-computer.jpg"
+                alt="SFD in Tehran"
+              />
+              <span class="cover"></span>
+              <h3>2021</h3>
+              <p>Bootcamp training 2021</p>
+            </a>
+            <a href="https://tehransfd.ir" class="program">
+              <img
+                src="./IMG/Past-Bootcamps/ju-training.jpg"
+                alt="SFD in Tehran"
+              />
+              <span class="cover"></span>
+              <h3>2020</h3>
+              <p>Bootcamp training 2020</p>
+            </a>
+          </div>
+        </div>
+      </section>
+      <section id="partners-sec" class="partners-about">
```

Comment:
hi @Deepakdanger. Thanks for the review, but why is it that one of the requirements says: "The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable."

I've seen so many projects for this milestone to just pay a look at their design and they implemented different images.
Is it truly necessary to change the logos, if so, could you tell me where Can I find the ones you want? because as there's no a Figma it's really though to find the resources, you know.

Reply:
Weldone with your implementation @SebastianSolisFenger , this is fine